packages/api/database/migrations/1603315501405_technology_comment_schema.js

```diff
@@ -0,0 +1,29 @@
+/** @type {import('@adonisjs/lucid/src/Schema')} */
+const Schema = use('Schema');
+
+class TechnologyCommentSchema extends Schema {
+	up() {
+		this.create('technology_comments', (table) => {
+			table.increments();
+			table
+				.integer('user_id')
+				.unsigned()
+				.references('id')
+				.inTable('users');
+			table
+				.integer('technology_id')
+				.unsigned()
+				.references('id')
+				.inTable('technologies')
+				.onDelete('cascade');
+			table.text('comment').notNullable();
```

Comment:
Certeza que é not null? No `TechnologyController` tem um if pra caso o usuário mande um comentário. Caso isso seja realmente not null, é bacana lançar uma exception lá no controller caso `comments` não seja enviado.

Reply:
@mateus4k é notNullable pq não existe uma situação onde se crie um registro nessa tabela com esse campo nulo, ou seja, se o usuário não enviar o comentário, nem sequer se cria, ou seja, ele nunca será nulo.